qipr_approver/approver/utils.py

```diff
@@ -209,3 +210,15 @@ def true_false_to_bool(true_false_string):
             return False
 
     return None
+
+def get_model_from_string(model_name):
```

Comment:
Not sure if this is the best / right way to do this. I had to do something similar for the bridge but I instead iterated over the app's models module or something. We should determine which is the best way because I'm really not sure.